frontend/lib/norent/letter-builder/rent-periods.tsx

```diff
@@ -26,9 +26,18 @@ function getCurrentRentNonpaymentPeriods(s: AllSessionInfo): string[] {
 function getRentNonpaymentChoices(
   periods: AllSessionInfo["norentAvailableRentPeriods"]
 ): DjangoChoices {
+  // LA residents' protections extend beyond those in other cities.
+  function addCaveatForLA(paymentDate: GraphQLDate) {
+    let caveat = "";
+    if (new Date(paymentDate) >= new Date("01 October 2021")) {
+      caveat = " (only for City of Los Angeles residents)";
```

Comment:
By the way, I think the easiest way to internationalize this snippet of text would be to wrap it in an `li18n` function. See how "Months you're missing rent payments" is internationalized on line 49 of this file for reference.